generator/test_case_generator.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type matrixRow struct {
-	TestDir             string `json:"testDir"`
+	TestDir             string `json:"test_dir"`
```

Comment:
I messed up the dir name. This fixes that.

Reply:
**Nitpick**: Other are not consistent with the format (e.g testType - test_type)